src/mk.sh

```diff
@@ -507,13 +507,13 @@ function kernel_deploy
 
     kernel_install "$reboot" "$name" "" "$target" "$remote"
     end=$(date +%s)
-    runtime=$(( runtime + (end - start) ))
+    runtime=$((runtime + (end - start)))
```

Comment:
I think the space in this expression adds to readability due to the ending token being `))`.
So it's not immediately obvious if it is a single `)` followed by a double `))` or three singles, or a double followed by a single.
Not sure what to prioritize here.